crates/next_version/src/version_increment.rs

```diff
@@ -136,3 +158,80 @@ impl VersionIncrement {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    // Returns true when a commit's custom type matches the provided regex
```

Comment:
This comment is not necessary.

Reply:
done 6d02777f8b00ba0ecad945dbd6391d532ee71093